server/services/omuCaselist.test.ts

```diff
@@ -0,0 +1,187 @@
+import { add, sub, format } from 'date-fns'
+import OmuCaselist from './omuCaselist'
+import LicenceStatus from '../enumeration/licenceStatus'
+import LicenceType from '../enumeration/licenceType'
+import Container from './container'
+
+describe('omu caselist', () => {
+  it('should return prison view case', () => {
+    const today = format(new Date(), 'yyyy-MM-dd')
+    const omuCaselist = createCaselist(LicenceStatus.SUBMITTED, today, '2022-10-19')
+
+    expect(omuCaselist.getPrisonView()).toEqual({
```

Comment:
think your getPrisonView should have 6 cases?
  prison status, in future
  prison status, in past
  probation status, in future
  probation status, in past
  Out of scope status, in future
  Out of scope status, in past


You're getProbationView could test the same 6?

Reply:
done